code_challenges/code_wars/trigrams.py

```diff
@@ -0,0 +1,19 @@
+# https://www.codewars.com/kata/55d8dc4c8e629e55dc000068
+
+
+def extract_trigrams(words):
+    stopping_point = len(words) - 2
+    if len(words) <= 3:
+        return ""
```

Comment:
Great. This is called a `guard clause` and is a great way to reduce complexity in code.  For example this could have been handled with nested if/else statements, but by returning early here we don't have to juggle this one off case below and the code is much simpler.